src/website/app/pages/Tokens/tokens.md

```diff
@@ -0,0 +1,58 @@
+Mineral UI distributes design tokens through the
+[mineral-ui-tokens][tokens-package] npm package. Mineral UI consumes these
```

Comment:
Maybe "...npm package and consumes...", to avoid starting the first two sentences with "Mineral UI"?